db/init/mysql/schema.sql

```diff
@@ -543,7 +543,15 @@ INSERT INTO `permission` VALUES ('1529402639376629896', '1346358560427216896', '
 INSERT INTO `permission` VALUES ('1534577122279825408', '1346358560427216896', '1534577121923309568', '2022-06-09 00:44:32', '2022-06-09 00:44:31');
 INSERT INTO `permission` VALUES ('1534585430587875328', '1346358560427216896', '1534585430311051264', '2022-06-09 01:17:33', '2022-06-09 01:17:32');
 INSERT INTO `permission` VALUES ('1534585531389583360', '1346358560427216896', '1534585531108564992', '2022-06-09 01:17:57', '2022-06-09 01:17:56');
-
+INSERT INTO `permission` VALUES ('1568879456483248858', '1346358560427216896', '1529402639284355100', '2022-09-18 21:45:10', '2022-09-18 21:45:12');
```

Comment:
not need  init permission and resource sql

Reply:
if dont init permission and resource sql , how can plugin display in here
![image](https://user-images.githubusercontent.com/88994283/191261260-41190531-0636-4eef-afd1-124455176c9c.png)

Reply:
There is a subsequent function that will come to initialize these permissions